hack/release/common.sh

```diff
@@ -123,16 +123,6 @@ cosignImages() {
         "${CONTROLLER_IMG}"
 }
 
-pullPrivateReplica(){
```

Comment:
There's still a call to this in `release()`, other than that looks good 👍

Reply:
Good call. I recalled I had originally scoped this into the change but look like somehow I managed to miss it.